src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -24,8 +24,12 @@ const actions: ActionTree<IEmptyState, IRootState> = {
       throw new ValidationException(`Unable to find resource for choice: ${resourceId}`)
     }
 
-    const choice = find(block.config.choices, (v) => v.prompt === resourceId) as IChoice
-    choice.name = resourceValue.value
+    // reactive way of editing choice.name
+    block.config.choices = block.config.choices.map(choice =>
+      (choice.prompt === resourceId
+        ? ({...choice, name: resourceValue.value})
+        : choice
+      ))
```

Comment:
without this change these choice labels didn't work (especially 'two')
<img width="352" alt="Screen Shot 2022-08-04 at 11 40 21 PM" src="https://user-images.githubusercontent.com/18130716/182957689-2cd761c6-12f5-4879-b56d-3c1406b7d996.png">